README.md

```diff
@@ -1,38 +1,343 @@
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| JAIL_ROLE          | The role that you give to members when you want to send them to your jail channel. this is used in the Auto Role module                           |
+| YOUNG_ACCOUNT_ROLE | This is the role you wish the bot to apply to accounts that are new to Discord if you have the `min Account Age` prop enabled on Auto Role module |
+
+Channel Settings:
+
+| Setting           | Description                                                                                                             |
+|-------------------|-------------------------------------------------------------------------------------------------------------------------|
+| LOG_CHANNEL       | The channel for the `Audit Log` module to post to                                                                       |
+| ADMIN_LOG_CHANNEL | the channel for the  `Admin Loggers` modules to post to                                                                 |
+| JAIL_CHANNEL      | the channel you use to `jail` members                                                                                   |
+| BIRTHDAY_CHANNEL  | Used in the `birthday` command to wish members happy birthday, if this setting is not set, the command will be disabled |
+
 ## <ins>Modules</ins>
 
-There are 12 main modules in this bot, each one has its own set of permissions as defined by the`CommandSecurityModels`
-database table.
-
-| Module | Description |
-| --- | ----------- |
-| Ages | commands to get ages of accounts and servers |
-| ResourceBanner | Commands deal with banning attachments, embeds and emojis from messages |
-| Mute | Commands to mute people from servers |
-| Username | Commands to set usernames for people|
-| Memes | Commands generate memes. All commands that are marked as type 'attachment' can take both an uploaded image (has to be jpg) OR a mention |
-| Bookmarks | Commands to manage bookmarks |
-| Help | Commands to display help and info |
-| messageschedule | Commands to schedule posts to channels|
-| Miscellaneous | Miscellaneous commands |
-| closeableModules | Commands to enable or disable modules and commands |
-| Music | Commands to play music from Youtube |
-| RoleJoiner | Commands to allow users to join vanity roles|
-| Settings | Commands to change internal settings of this bot |
-
-you may use the`~help`command for more info about each command and how to call each command complete with mandatory
-arguments and examples.
+All modules in this application are defined by implementing the `ICloseableModule` interface. this applies to events
+only, not commands. Commands are defined in the own entity set contained in categories.
+
+### <ins>events</ins>
+
+#### Audit Logger
+
+The Audit logger is a non-admin styled logger designed to log quick events like when a member leaves, joins, kicked and
```

Comment:
It's a bit of a mouth full, How about:
"The Audit logger is a non-admin styled logger designed to log quick events like when a member joins, leaves, is kicked or banned (along with by whom) and when an Auto Mod event is triggered."